There's still plenty of `using namespace` in other places. Should we use this PR to get rid of those too?

Appveyor seems to have some problems:
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
ICONV_LIBRARY
    linked by target "cmTC_250f2" in directory C:/projects/exiv2-wutfp/build/CMakeFiles/CMakeTmp
CMake Error at C:/Program Files (x86)/CMake/share/cmake-3.11/Modules/CheckIncludeFile.cmake:60 (try_compile):
  Failed to configure test project build system.
Call Stack (most recent call first):
  config/generateConfigFile.cmake:35 (check_include_file)
  CMakeLists.txt:83 (include)
-- Configuring incomplete, errors occurred!
```
I think it complained about this too in #264, so it is quite surely unrelated to this PR.

--- a/include/exiv2/asfvideo.hpp
+++ b/include/exiv2/asfvideo.hpp
@@ -33,10 +33,6 @@
 #include "exif.hpp"
 #include "image.hpp"
 
-// *****************************************************************************
-// namespace extensions
-using namespace Exiv2::Internal;
-
 namespace Exiv2 {
 
 // *****************************************************************************
@@ -94,7 +90,7 @@ namespace Exiv2 {
           @param tv Pointer to current tag,
           @param size Size of the data block used to store Tag Information.
          */
-        void tagDecoder(const TagVocabulary* tv, uint64_t size);
+        void tagDecoder(const Internal::TagVocabulary* tv, uint64_t size);
         /*!
           @brief Interpret File_Properties tag information, and save it in
               the respective XMP container.

